.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -116,5 +116,7 @@ body:
           required: true
         - label: I filled out all of the requested information in this issue properly.
           required: true
-        - label: The issue is related solely to the ReVanced Manager and not related to patching errors or patches
+        - label: The issue is related solely to the ReVanced Manager and not related to patching errors not caused by issues of the ReVanced Manager.

```

Comment:
This implies the user is aware of whether or not errors are caused by ReVanced Manager or ReVanced Patcher

Reply:
I don't think there could be a sure way for the user to know what their issue is related to unless they understand the context and meaning of the patching error.

But before the user was also expected to know what their issue related to. But before it didn't say this explicitly, which led me, and likely other users, whose issues were indeed related to issues with the Manager to report in the patcher and patches repo instead.

Reply:
There's not much we can do about this. The end user is forced to differentiate between those errors. The suggested sentence is also very confusing, I don't understand what it means

Reply:
Is this better?

"The issue is not related to patching errors not caused by a ReVanced Manager issue.

The issue is related solely to the ReVanced Manager and not related to issues with patches."

I can't think of a way to avoid having a double negative in the first sentence.

Reply:
> The issue is related solely to the ReVanced Manager and not related to issues with patches.

How about:


The issue is solely related to ReVanced Manager and not patching errors or patches

Reply:
That's what it was lol. But the issue is that it implies that all patching errors do not belong in this repo, as discussed.

Reply:
Correct, all patching errors do not belong in this repo. Only errors caused by ReVanced Manager.

Reply:
But it doesn't suggest that there is a difference between a "patching error" and an error (while patching) caused by the manager.

Without explicitly saying that a differentiation can be made the user will never know. It's best to inform the user, even at the cost of a slightly difficult-to-read sentence, rather than scare them away due to a lack of understanding.

Reply:
> But it doesn't suggest that there is a difference between a "patching error" and an error (while patching) caused by the manager.

I can not see this in your suggestion either though.

Reply:
Sorry. It isn't explicit, but it is heavily implied.

Reply:
I have a more challenging time understanding the suggestion. The implication is present in the current text as well.

Reply:
It is not. 

"solely to the ReVanced Manager and not related to patching errors or patches." implies that the issue must be something not related AT ALL to patching errors.

Reply:
There's no right way to word this, there's really absolutely no way for the user to figure out at which part of the chain the error occured

Reply:
Depending on context (the error message, patches applied, application) and knowledge of the user, they may be able to discern what is happening. For example, if the same error is given for all applications the issue is likely with the Manager.

Reply:
> It says nothing about an exeption for patching errors that are caused by the manager.

It does, implicitly by saying "solely to ReVanced Manager", that implicitly includes errors caused by ReVanced Manager.

Reply:
Last attempt

It is difficult to deduct that.

What about, "The issue is related solely to the ReVanced Manager and not related to other patching errors or patches."

Reply:
There is no difference or any additional benefit to the current sentence.